config/environments/production.rb

```diff
@@ -6,12 +6,12 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.smtp_settings = { # WARNING do not let standardrb linter change this block, it breaks
-      :address => 'smtp-relay.sendinblue.com',
```

Comment:
These lines cannot be changed, please see the comment. `# WARNING do not let standardrb linter change this block, it breaks`